src/platform/user/profile/vet360/containers/AddressValidationModal.jsx

```diff
@@ -233,24 +223,30 @@ class AddressValidationModal extends React.Component {
 }
 
 const mapStateToProps = state => {
```

Comment:
With so many props being used in this component, could we add their respective PropTypes to it?